components/raftstore/src/store/peer.rs

```diff
@@ -2324,8 +2324,22 @@ where
                         cp
                     ));
                 }
+                (ConfChangeType::AddNode, PeerRole::Voter) => {
+                    let pr = match current_progress.get(peer.get_id()) {
+                        Some(pr) => pr,
+                        None => {
+                            return Err(box_err!("{} invalid conf change request: {:?}, voter should be added as learner first.", self.tag, cp));
+                        }
+                    };
+                    if pr.matched == 0 {
```

Comment:
If the peer hasn't send any messages to the leader  in the leader's current term, `pr.matched` should be  0. However the peer can have been already initialized.

Reply:
Yes. But it seems there is no better way to recognize this situation.

Reply:
How about comment it?